deploy/v2.0.0-alpha.1/rbac-csi-azuredisk-node.yaml

```diff
@@ -18,12 +18,24 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["coordination.k8s.io"]
```

Comment:
The `coordination.k8s.io/leases` permissions are missing from the Helm charts. Are they needed? (I assume so since they are for leader election, right?)

Reply:
I didn't mean to add this node's cluster role; that's why it's missing from the helm chart. They are only needed for controller cluster role for leader election process as you mentioned. node plugin doesn't need it as it doesn't go through leader election. Fixed. Thanks for pointing this out!